api/src/main/java/org/jboss/arquillian/graphene/context/GrapheneContext.java

```diff
@@ -54,7 +54,7 @@ public abstract class GrapheneContext {
      * @param interfaces interfaces which should be implemented by the returned {@link WebDriver}
      * @return proxy for the {@link WebDriver} held in the context
      */
-    public abstract WebDriver getWebDriver(Class<?>... interfaces);
+    public abstract WebDriver getWebDriver(boolean dontProxy, Class<?>... interfaces);
```

Comment:
In order to avoid touching existing methods where we don't need the override.

```suggestion
    public abstract WebDriver getWebDriver(Class<?>... interfaces);
    
    public WebDriver getWebDriver(boolean noProxy, Class<?>... interfaces){
        return getWebDriver(false, interfaces);
    }
```

Reply:
The `dontProxy` stuff is scaffolding to help me debug the proxies. I don't think this will remain in the final code.
However, your suggestion is great and will alleviate the need to change tests.
Thanks!